### ISSUE TITLE
if save can support ,safe_serialization=False

### ISSUE THREAD

**Issue by NLPJCL** [UKPLab/sentence-transformers#2533]
The current save function cannot be set safe_serialization。but defalut safe_serialization=True。thank！

**Comment by tomaarsen** [comment]
Hello!

You're right, it would be nice if users could set `safe_serialization` so that they can save a `pytorch_model.bin` if they prefer. 

- Tom Aarsen

**Comment by CKeibel** [comment]
Hi @tomaarsen ! 
If no one is working on this issue yet, could I work on this to get started with contributing?

**Comment by tomaarsen** [comment]
That would be great!
I think we would need to wrap [this line](https://github.com/UKPLab/sentence-transformers/blob/66e0ee30843dd411c64f37f65447bb38c7bf857a/sentence_transformers/SentenceTransformer.py#L619) with `if isinstance(module, Transformer):` and then pass the `safe_serialization` down to the Transformer, while all other modules still stay as `module.save(model_path)`. 

- Tom Aarsen

**Comment by CKeibel** [comment]
Great, thanks for the tip. I'll have a look at it.

**Comment by llinauer** [comment]
This issue was apparently solved in https://github.com/UKPLab/sentence-transformers/pull/2542

**Comment by tomaarsen** [comment]
You're right, thanks for mentioning. I forgot to link the issue to the pull request! I'll close this.

- Tom Aarsen
